onnxruntime/core/mlas/lib/mlasi.h

```diff
@@ -774,7 +774,7 @@ MlasGetMaximumThreadCount(
     return 1;
 #endif
 #else
-	return onnxruntime::concurrency::ThreadPool::NumThreads(ThreadPool);
+	return onnxruntime::concurrency::ThreadPool::DegreeOfParallelism(ThreadPool);
```

Comment:
If you change any of the other nits, can you also change the tab on this line to 4 spaces? This was a problem from an earlier PR.

Reply:
Thanks, I updated all of these.  Will push another small commit with just these edits.